src/index.js

```diff
@@ -3,16 +3,18 @@
  * @class AddressAutocomplete
  * @author Deric Cain <deric.cain@gmail.com>
  * @contributor Braunson Yager <braunson@gmail.com>
+ * @contributor Jonathan Sardo <sardoj@gmail.com>
  */
 export default class AddressAutocomplete {
   /**
    * Creates an instance of AddressAutocomplete.
    * @param {string} element - This should be in the form of either '.address' or '#address'
+   * @param {object} options - This object contains options to add to the API call
    * @param {function} callback - This callback will have the result passed as the first param
    * @throws Error - If we don't have a valid element
    * @memberof AddressAutocomplete
    */
-  constructor(element, callback) {
+  constructor(element, options, callback) {
```

Comment:
Change the name of `options` to something like `optionsOrCallback` since it could be either. Make `callback` have a default value of `null` like: `callback = null` so we do not have to do a check later.